[32mINFO    [0m root:test_02_usersync.py:110 ==========开始执行用例：usersync_syncConfig_sync_01==========
[32mINFO    [0m root:runcase.py:31 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
[32mINFO    [0m root:handleyaml.py:94 Yaml测试用例基础架构检查通过
[32mINFO    [0m root:runcase.py:31 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
[32mINFO    [0m root:handleyaml.py:94 Yaml测试用例基础架构检查通过
[32mINFO    [0m root:runcase.py:50 重组后的用例信息：{'name': '查询所有同步配置', 'description': '查询所有同步配置', 'validate': {'code': 200, 'response': {'statusCode': 0, 'msg': 'success'}}, 'epic': '用户管理', 'feature': '用户同步管理-用户同步', 'story': '查询所有同步配置接口', 'base_url': 'https://192.168.148.174:31000', 'request': {'method': 'GET', 'address': '/api/usersync/v1/syncConfig', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6Ijc2ZDQ0NmFlLTdkMGEtNDVhYy04NzE0LTgxNmQzMzRjYzg4ZSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3NmQ0NDZhZS03ZDBhLTQ1YWMtODcxNC04MTZkMzM0Y2M4OGUiLCJleHAiOjE2ODgxMjIyMDEsImlhdCI6MTY4Nzk0MjIwMX0.FWYPT5nPgOdBtOT0wGRVZNVTB6oggV1ehXLgP2gppALL2Djxp_46T_KhaOizqJlxUPAYRWJs7nKHiU-ug2qGKA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}}, 'caseid': 'usersync_syncConfig_alllist_01'}
[32mINFO    [0m root:runcase.py:81 请求地址：https://192.168.148.174:31000/api/usersync/v1/syncConfig
[32mINFO    [0m root:runcase.py:82 请求方法：GET
[32mINFO    [0m root:runcase.py:83 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6Ijc2ZDQ0NmFlLTdkMGEtNDVhYy04NzE0LTgxNmQzMzRjYzg4ZSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3NmQ0NDZhZS03ZDBhLTQ1YWMtODcxNC04MTZkMzM0Y2M4OGUiLCJleHAiOjE2ODgxMjIyMDEsImlhdCI6MTY4Nzk0MjIwMX0.FWYPT5nPgOdBtOT0wGRVZNVTB6oggV1ehXLgP2gppALL2Djxp_46T_KhaOizqJlxUPAYRWJs7nKHiU-ug2qGKA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "GET /api/usersync/v1/syncConfig HTTP/1.1" 200 267
[32mINFO    [0m root:runcase.py:166 请求接口结果： {'statusCode': 0, 'msg': 'success', 'data': [{'id': 42, 'modifyTime': '2023-06-28 16:53:03', 'userSourceId': '55', 'jobTime': '00 39 15 10 * ?', 'isIncrement': 1, 'threshold': 0, 'taskSavePeriod': 180, 'syncName': '自动化测试同步配置修改', 'syncType': '0', 'allowUpdate': False}]}
[32mINFO    [0m root:checkresult.py:152 code断言通过, 期望状态码:200, 实际状态码:200
[32mINFO    [0m root:checkresult.py:87 返回结果断言通过, 期望返回值:{'statusCode': 0, 'msg': 'success'}, 实际返回值:{'statusCode': 0, 'msg': 'success', 'data': [{'id': 42, 'modifyTime': '2023-06-28 16:53:03', 'userSourceId': '55', 'jobTime': '00 39 15 10 * ?', 'isIncrement': 1, 'threshold': 0, 'taskSavePeriod': 180, 'syncName': '自动化测试同步配置修改', 'syncType': '0', 'allowUpdate': False}]}
[32mINFO    [0m root:runcase.py:61 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-
[32mINFO    [0m root:runcase.py:50 重组后的用例信息：{'name': 'LADP域立即同步', 'description': '同步LADP数据源', 'request': {'data': {'syncConfigId': '42'}, 'method': 'PUT', 'address': '/api/usersync/v1/sync', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6Ijc2ZDQ0NmFlLTdkMGEtNDVhYy04NzE0LTgxNmQzMzRjYzg4ZSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3NmQ0NDZhZS03ZDBhLTQ1YWMtODcxNC04MTZkMzM0Y2M4OGUiLCJleHAiOjE2ODgxMjIyMDEsImlhdCI6MTY4Nzk0MjIwMX0.FWYPT5nPgOdBtOT0wGRVZNVTB6oggV1ehXLgP2gppALL2Djxp_46T_KhaOizqJlxUPAYRWJs7nKHiU-ug2qGKA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}}, 'relevance': [{'caseid': 'usersync_syncConfig_alllist_01', 'response': [{'value': '$.data[0].id', 'name': 'syncConfigId'}]}], 'validate': {'code': 200, 'response': {'statusCode': 0, 'msg': 'success'}}, 'epic': '用户管理', 'feature': '用户同步管理-用户同步', 'story': '立即同步接口', 'base_url': 'https://192.168.148.174:31000', 'caseid': 'usersync_syncConfig_sync_01'}
[32mINFO    [0m root:runcase.py:81 请求地址：https://192.168.148.174:31000/api/usersync/v1/sync
[32mINFO    [0m root:runcase.py:82 请求方法：PUT
[32mINFO    [0m root:runcase.py:83 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6Ijc2ZDQ0NmFlLTdkMGEtNDVhYy04NzE0LTgxNmQzMzRjYzg4ZSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3NmQ0NDZhZS03ZDBhLTQ1YWMtODcxNC04MTZkMzM0Y2M4OGUiLCJleHAiOjE2ODgxMjIyMDEsImlhdCI6MTY4Nzk0MjIwMX0.FWYPT5nPgOdBtOT0wGRVZNVTB6oggV1ehXLgP2gppALL2Djxp_46T_KhaOizqJlxUPAYRWJs7nKHiU-ug2qGKA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}
[32mINFO    [0m root:runcase.py:93 请求参数：{'syncConfigId': '42'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "PUT /api/usersync/v1/sync HTTP/1.1" 200 32
[32mINFO    [0m root:runcase.py:183 请求接口结果： {'statusCode': 0, 'msg': 'success'}
[32mINFO    [0m root:checkresult.py:152 code断言通过, 期望状态码:200, 实际状态码:200
[32mINFO    [0m root:checkresult.py:87 返回结果断言通过, 期望返回值:{'statusCode': 0, 'msg': 'success'}, 实际返回值:{'statusCode': 0, 'msg': 'success'}
[32mINFO    [0m root:runcase.py:61 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-